/home/johwar/pentaho6/data-integration
DEBUG: Found JRE at the current folder
DEBUG: _PENTAHO_JAVA_HOME=/home/johwar/pentaho6/data-integration/jre
DEBUG: _PENTAHO_JAVA=/home/johwar/pentaho6/data-integration/jre/bin/java
DEBUG: PENTAHO_INSTALLED_LICENSE_PATH=
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
21:11:45,784 INFO  [KarafBoot] Checking to see if org.pentaho.clean.karaf.cache is enabled
21:11:45,859 INFO  [KarafInstance] 
*******************************************************************************
*** Karaf Instance Number: 1 at /home/johwar/pentaho6/data-integration-6.1. ***
***   0.6-242/./system/karaf/caches/pan/data-1                              ***
*** Karaf Port:8802                                                         ***
*** OSGI Service Port:9051                                                  ***
*******************************************************************************
Nov 07, 2016 9:11:46 PM org.apache.karaf.main.Main$KarafLockCallback lockAquired
INFO: Lock acquired. Setting startlevel to 100
2016/11/07 21:11:47 - General - Logging plugin type found with ID: CheckpointLogTable
Refreshing GOE props...
2016-11-07 21:11:48 weka.gui.GenericObjectEditor registerEditors
INFO: ---Registering Weka Editors---
Trying to add database driver (JDBC): RmiJdbc.RJDriver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): jdbc.idbDriver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): org.gjt.mm.mysql.Driver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): com.mckoi.JDBCDriver - Warning, not in CLASSPATH?
[KnowledgeFlow] Loading properties and plugins...
[KnowledgeFlow] Initializing KF...
Refreshing GOE props...
2016-11-07 21:11:49 weka.gui.GenericObjectEditor registerEditors
INFO: ---Registering Weka Editors---
Nov 07, 2016 9:11:50 PM org.pentaho.caching.impl.PentahoCacheManagerFactory$RegistrationHandler$1 onSuccess
INFO: New Caching Service registered
Trying to add database driver (JDBC): RmiJdbc.RJDriver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): jdbc.idbDriver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): org.gjt.mm.mysql.Driver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): com.mckoi.JDBCDriver - Warning, not in CLASSPATH?
[KnowledgeFlow] Loading properties and plugins...
[KnowledgeFlow] Initializing KF...
Refreshing GOE props...
2016-11-07 21:11:51 weka.gui.GenericObjectEditor registerEditors
INFO: ---Registering Weka Editors---
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/johwar/pentaho6/data-integration-6.1.0.6-242/launcher/../lib/slf4j-log4j12-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/johwar/pentaho6/data-integration-6.1.0.6-242/plugins/pentaho-big-data-plugin/lib/slf4j-log4j12-1.7.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
Trying to add database driver (JDBC): RmiJdbc.RJDriver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): jdbc.idbDriver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): org.gjt.mm.mysql.Driver - Warning, not in CLASSPATH?
Trying to add database driver (JDBC): com.mckoi.JDBCDriver - Warning, not in CLASSPATH?
[KnowledgeFlow] Loading properties and plugins...
[KnowledgeFlow] Initializing KF...
2016/11/07 21:11:53 - Pan - Start of run.
2016/11/07 21:11:53 - test_jndi_both - Dispatching started for transformation [test_jndi_both]
org.pentaho.di.core.exception.KettleDatabaseException: 
An error occurred executing SQL: 
SELECT n FROM foo
ERROR: relation "foo" does not exist
  Position: 15
	at org.pentaho.di.core.database.Database.openQuery(Database.java:1716)
	at org.pentaho.di.trans.steps.tableinput.TableInput.doQuery(TableInput.java:224)
	at org.pentaho.di.trans.steps.tableinput.TableInput.processRow(TableInput.java:138)
	at org.pentaho.di.trans.step.RunThread.run(RunThread.java:62)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "foo" does not exist
  Position: 15
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2198)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1927)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:561)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:405)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:285)
	at org.apache.commons.dbcp.DelegatingStatement.executeQuery(DelegatingStatement.java:208)
	at org.apache.commons.dbcp.DelegatingStatement.executeQuery(DelegatingStatement.java:208)
	at org.pentaho.di.core.database.Database.openQuery(Database.java:1705)
	... 4 more
2016/11/07 21:11:53 - Table input JW_TEST1.0 - ERROR (version 6.1.0.6-242, build 1 from 2016-09-20 13.19.30 by buildguy) : Unexpected error
2016/11/07 21:11:53 - Table input JW_TEST1.0 - ERROR (version 6.1.0.6-242, build 1 from 2016-09-20 13.19.30 by buildguy) : org.pentaho.di.core.exception.KettleDatabaseException: 
2016/11/07 21:11:53 - Table input JW_TEST1.0 - An error occurred executing SQL: 
2016/11/07 21:11:53 - Table input JW_TEST1.0 - SELECT n FROM foo
2016/11/07 21:11:53 - Table input JW_TEST1.0 - ERROR: relation "foo" does not exist
2016/11/07 21:11:53 - Table input JW_TEST1.0 -   Position: 15
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.pentaho.di.core.database.Database.openQuery(Database.java:1716)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.pentaho.di.trans.steps.tableinput.TableInput.doQuery(TableInput.java:224)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.pentaho.di.trans.steps.tableinput.TableInput.processRow(TableInput.java:138)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.pentaho.di.trans.step.RunThread.run(RunThread.java:62)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at java.lang.Thread.run(Thread.java:745)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - Caused by: org.postgresql.util.PSQLException: ERROR: relation "foo" does not exist
2016/11/07 21:11:53 - Table input JW_TEST1.0 -   Position: 15
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2198)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1927)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:561)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:405)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:285)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.apache.commons.dbcp.DelegatingStatement.executeQuery(DelegatingStatement.java:208)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.apache.commons.dbcp.DelegatingStatement.executeQuery(DelegatingStatement.java:208)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	at org.pentaho.di.core.database.Database.openQuery(Database.java:1705)
2016/11/07 21:11:53 - Table input JW_TEST1.0 - 	... 4 more
child index = 1, logging object : org.pentaho.di.core.logging.LoggingObject@2cb29046 parent=4c283b44-95e9-4777-bbbe-dfda72f1353a
2016/11/07 21:11:53 - Table input JW_TEST1.0 - Finished reading query, closing connection.
2016/11/07 21:11:53 - Table input JW_TEST2.0 - Finished reading query, closing connection.
2016/11/07 21:11:53 - Table input JW_TEST1.0 - Finished processing (I=0, O=0, R=0, W=0, U=0, E=1)
2016/11/07 21:11:53 - Table input JW_TEST2.0 - Finished processing (I=2, O=0, R=0, W=0, U=0, E=0)
2016/11/07 21:11:53 - test_jndi_both - Transformation detected one or more steps with errors.
2016/11/07 21:11:53 - test_jndi_both - Transformation is killing the other steps!
2016/11/07 21:11:53 - Pan - Finished!
2016/11/07 21:11:53 - Pan - Start=2016/11/07 21:11:53.010, Stop=2016/11/07 21:11:53.502
2016/11/07 21:11:53 - Pan - Processing ended after 0 seconds.
